src/components/NFTEditionSelect.vue

```diff
@@ -9,12 +9,13 @@
           v-for="(item, index) in items"
           :key="index"
           :name="item.name"
+          :currency="item.currency"
           :price-label="item.priceLabel"
+          :allow-custom-price="item.isAllowCustomPrice"
```

Comment:
Is this prop `allow-custom-price` still used?

Reply:
🔥 Remove unused code